keyboards/themadnoodle/noodlepad/readme.md

```diff
@@ -0,0 +1,17 @@
+# Noodle Pad v1 Keypad with Encoder
```

Comment:
This file needs updating, still contains images and references to ncc1701kb.

Reply:
The original must've gotten lost during all of the back and forths. 
It's now fixed.